README.md

```diff
@@ -1,37 +1,175 @@
-# NAME OF PROJECT
+# Poll Tracker
 
 ## Project Objectives
 
 ```zsh
 git checkout -b workOnANewBranch
 ```
 
-* Essential Goals:
+### Essential Goals
 
-  *
+* Project should contain 3 major sections:
+  
+1. Input form section -
 
-* Stretch Goals:
+    *Users enter text to the input elements in this form to submit data for the following:*
 
-  *
+    * Question Input
 
-### HTML SETUP
+    * Option A Input
+
+    * Option B Input
+
+    *User submit event for form data:*
+
+    * Submit button
+
+1. Current poll Section -
+
+    **Elements in this section should not display text content until form data submit event.**
+
+    *On form data submit event, display form data for the following:*
+
+    * Question Input form data
+
+    * Option A Input form data
+
+    * Option B Input form data
+
+    *User click event for the following button elements:*
+
+    * Option A increment
+
+    * Option A decrement
+
+    * Option B increment
+
+    * Option B decrement
+
+    *User click event should update state for the following text elements:*
+
+    * Option A results text element
+
+    * Option B results text element
+
+        *User click event should clear section data and update state for Past Results Section data*
+
+    * Close Poll submit
+
+1. Past Results Section -
+
+    * Past Results text element
+
+    * Flexible container for *Closed Poll* elements
+
+    **Elements in this section should not display until Close Poll Submit click  event.**
+
+    *note*: closed poll item elements will be created in JSON.
+
+    * Container element for closed poll items
+
+    * Past Result Question text element
+
+    * Past Result Option A text element
+
+    * Past Result Option B text Element
+
+    * Past Result Option A Result text element
+
+    * Past Result Option B Result text element
+
+### Stretch Goals
+
+* **tbd**
+
+## HTML SETUP
+
+* Input form section -
+
+  1. form element to contain:
+
+      * 3 text input element for user to submit 'question', 'answer a', answer b'
+
+      * button element for user to submit form data to be applied to current poll section
```

Comment:
It looks like you took some time to plan this out.  It shows in your code.  Nice work!